azure/resource_groups/app/template.json

```diff
@@ -119,24 +119,23 @@
 
     "applicationInsightsDeploymentName": "[concat(parameters('resourceNamePrefix'), '-application-insights')]",
     "appServiceCertificateDeploymentName": "[concat(parameters('resourceNamePrefix'), '-app-service-certificate')]",
+    "appServiceDeploymentName": "[concat(parameters('resourceNamePrefix'), '-app-service-web')]",
+    "appServiceWorkerDeploymentName": "[concat(parameters('resourceNamePrefix'), '-app-service-worker')]",
     "appServicePlanDeploymentName": "[concat(parameters('resourceNamePrefix'), '-app-service-plan')]",
+    "appServicePlanWorkerDeploymentName": "[concat(parameters('resourceNamePrefix'), '-worker-app-service-plan')]",
```

Comment:
Can we be consitent with these deployment names, please? Suggest `-web-*` and `-worker-*`.